packages/nimble-components/src/card/template.ts

```diff
@@ -2,7 +2,8 @@ import { html } from '@microsoft/fast-element';
 import type { Card } from '.';
 
 export const template = html<Card>`
-    <section aria-labelledby="title-slot">
+    <!-- Explicitly set role to work around Lighthouse error. See https://github.com/ni/nimble/issues/1650. -->
```

Comment:
On a phone right now but double check the comment style in other templates

Reply:
Updated the comment to use a template literal.